typescript/phoenix-sdk/src/utils/market.ts

```diff
@@ -547,6 +547,10 @@ export function getMarketL3Book(
           toNum(restingOrder.traderIndex)
         ),
         orderSequenceNumber: getUiOrderSequenceNumber(orderId),
+        lastValidSlot: toBN(restingOrder.lastValidSlot),
```

Comment:
what is the original type? do we need to make it a BN?

Reply:
`restingOrder.lastValidSlot` is a `beet.bignum`. This is being treated the same as `restingOrder.numBaseLots` which is a `beet.bignum` that we call `toBn(...)` on